sdk/keyvault/Azure.Security.KeyVault.Keys/src/DeletedKey.cs

```diff
@@ -31,14 +31,6 @@ public class DeletedKey : Key
 
         internal DeletedKey(string name) : base(name) { }
 
-        public DeletedKey (string name, string recoveryId, DateTimeOffset? deletedDate, DateTimeOffset? scheduledPurge)
```

Comment:
Was this intentional? It will be a breaking change from the last preview so just wanted to ensure.

Reply:
I noticed that it wasn't used and after looking more into it, the user shouldn't instantiate a `DeleteKey`. It is an object always returned/created by the service.
It feels more like a cleanup.